src/foam/u2/detail/SectionView.js

```diff
@@ -53,13 +53,21 @@ foam.CLASS({
 
       self
         .addClass(self.myClass())
-        .add(self.slot(function(section, showTitle, section$title) {
+        .add(self.slot(function(section, showTitle, section$title, section$subTitle) {
           if ( ! section ) return;
           return self.Rows.create()
             .show(section.createIsAvailableFor(self.data$))
             .callIf(showTitle && section$title, function() {
               this.start('h2').add(section$title).end();
             })
+            .callIf(section$subTitle, function() {
+              this.start().style({
+                'color': '#8e9090',
```

Comment:
Can you check for other colour assignment, I believe it's /* NAME */, fallback   or something to this affect.  With colours taken from theme.

Reply:
Macro reference example `color: /*%BLACK%*/ #1e1f21` I think you'll be looking for  `/*%GREY4%*/` for the subtitle.